v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -62,9 +62,10 @@ const responseBuilder = async (metadata, message, { Config }, payload) => {
   };
   response.params = { event, customerId: filteredCustomerId };
   if (Config.subAccount)
-    if (Config.loginCustomerId)
-      response.headers["login-customer-id"] = Config.loginCustomerId;
-    else
+    if (Config.loginCustomerId) {
+      const filteredLoginCustomerId = Config.loginCustomerId.replace(/-/g, "");
```

Comment:
Refactor this to a util method for this destination. I see it's already used in other places.